js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -2911,7 +2911,9 @@ module.exports = {
                         editingController.closeEditCell();
                     }
 
-                    return eventName === startEditAction && allowEditing && editingController.editCell(e.rowIndex, columnIndex) || editingController.isEditRow(e.rowIndex);
+                    if(allowEditing && eventName === startEditAction) {
```

Comment:
this could be in a separate variable, but it doesn't matter

Reply:
How can we name this variable? Seems there's no need to extract the condition in this case.